whelktool/scripts/syslib/2021/01/Udix_remove_print.groovy

```diff
@@ -0,0 +1,22 @@
+PrintWriter failedUpdating = getReportWriter("failed-updating")
+PrintWriter scheduledForUpdating = getReportWriter("scheduled-updates")
+
+File HoldIDs = new File(scriptDir, "Udix_ID2.txt")
+
+List propertiesToCheck = ["associatedMedia", "marc:hasTextualHoldingsBasicBibliographicUnit"]
+
+selectByIds( HoldIDs.readLines() )  { hold ->
+def holds = hold.graph[1]
```

Comment:
Nitpick: the use of hold (an individual hold) vs holds (here a part of that individual hold) is slightly confusing. Maybe "holds" => "item" would be better. (Doesn't matter for a one-off script, but for the future :smile:).

Reply:
> Looks good then! Maybe you should check also if the property is left empty after removing the "Externt magasin" object, and if that's the case, remove the property from graph[1]. If you think there are cases where associatedMedia/marc:hasTextualHoldingsBasicBibliographicUnit contains only one ("Externt magasin") object?

In general this is probably a very good idea, but the case here is that if there is only one object in associatedMedia/marc:hasTextualHoldingsBasicBibliographicUnit ("Externt magasin") the entire holdings record should be removed. That will be done separately (i.e. registervårdsklienten).